src/controllers/tripRequestController.js

```diff
@@ -238,3 +239,44 @@ export const deleteTripRequest = async (req, res) => {
     return res.status(500).json(error.message);
   }
 };
+
+export const mostTavelledDestinations = catchAsync(async (req, res, next) => {
```

Comment:
i think you could use one query to get all these data

Reply:
thank you.